src/styles/sidebar/components/group-list-out-of-scope-item.scss

```diff
@@ -0,0 +1,46 @@
+/* The group-list out of scope item. */
+@import "./group-list-item";
+@import "./group-list";
```

Comment:
`@import` in SASS causes a file to be included _each_ time it is used because it is a crude textual inclusion. This will cause the `group-list` CSS to be included twice in the final bundle.

The current rule of thumb is that things that will be used in multiple components (variables, mixins, functions) need to be put in separate files. Currently we have a single [variables.scss file](https://github.com/hypothesis/client/blob/d0ed54a5b3bbc45e6ccca67f416e5517a5fd2674/src/styles/variables.scss#L123) for this. We could also create a more narrowly scoped one in `src/styles/sidebar/variables.scss` in future.

There is some possibility that SASS might actually get a [proper module system](http://sass.logdown.com/posts/502818-request-for-comments-module-system-proposal) later this year, but that is still being worked on.

Reply:
This is one of the minor tweaks I wanted to make after yesterday.